--- a/lib/FusionInventory/Agent/Task/Inventory/Virtualization/Virtuozzo.pm
+++ b/lib/FusionInventory/Agent/Task/Inventory/Virtualization/Virtuozzo.pm
@@ -7,11 +7,7 @@ use FusionInventory::Agent::Tools;
 
 sub isEnabled {
     # Avoid duplicated entry with libvirt
-    return if getFirstMatch(
-        command => "virsh uri",
-        pattern => "openvz:///system"
-    );
-
+    return if canRun('virsh');
 
     return canRun('vzlist');
 }
