--- a/lib/FusionInventory/Agent/Task/Inventory/Generic/Dmidecode.pm
+++ b/lib/FusionInventory/Agent/Task/Inventory/Generic/Dmidecode.pm
@@ -10,8 +10,18 @@ use FusionInventory::Agent::Tools::Generic;
 
 sub isEnabled {
 
-    return unless getDmidecodeInfos();
-
+    # don't run dmidecode on Win2003
+    # http://forge.fusioninventory.org/issues/379
+    if ($OSNAME eq 'MSWin32') {
+        Win32->require();
+        my @osver = Win32::GetOSVersion();
+        return if
+            $osver[4] == 2 &&
+            $osver[1] == 5 &&
+            $osver[2] == 2;
+    }
+
+    return canRun('dmidecode');
 }
 
 sub doInventory {}
