--- mdb.c	Mon Nov 30 18:02:00 2015
+++ mdb.c	Wed Dec  9 09:32:59 2015
@@ -118,7 +118,7 @@
 #if defined(__APPLE__) || defined (BSD)
 # define MDB_USE_POSIX_SEM	1
 # define MDB_FDATASYNC		fsync
-#elif defined(ANDROID)
+#elif defined(ANDROID) || defined(__sun)
 # define MDB_FDATASYNC		fsync
 #endif
 
@@ -257,7 +257,7 @@
 # else
 #  define MDB_USE_ROBUST	1
 /* glibc < 2.10 only provided _np API */
-#  if defined(__GLIBC__) && GLIBC_VER < 0x02000a
+#  if defined(__sun) || (defined(__GLIBC__) && GLIBC_VER < 0x02000a)
 #   define PTHREAD_MUTEX_ROBUST	PTHREAD_MUTEX_ROBUST_NP
 #   define pthread_mutexattr_setrobust(attr, flag)	pthread_mutexattr_setrobust_np(attr, flag)
 #   define pthread_mutex_consistent(mutex)	pthread_mutex_consistent_np(mutex)
